validator/client/sync_committee.go

```diff
@@ -0,0 +1,68 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	validatorpb "github.com/prysmaticlabs/prysm/proto/validator/accounts/v2"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/traceutil"
+	"go.opencensus.io/trace"
+)
+
+// SubmitSyncCommitteeMessage submits the sync committee message to the beacon chain.
+func (v *validator) SubmitSyncCommitteeMessage(ctx context.Context, slot types.Slot, pubKey [48]byte) {
+	ctx, span := trace.StartSpan(ctx, "validator.SubmitSyncCommitteeMessage")
+	defer span.End()
+	span.AddAttributes(trace.StringAttribute("validator", fmt.Sprintf("%#x", pubKey)))
+
+	v.waitOneThirdOrValidBlock(ctx, slot)
+
+	res, err := v.validatorClient.GetSyncMessageBlockRoot(ctx, &ethpb.SyncMessageBlockRootRequest{Slot: slot})
```

Comment:
```suggestion
	res, err := v.validatorClient.GetSyncMessageBlockRoot(ctx, &ethpb.SyncMessageBlockRootRequest{Slot: slot -1})
```
As discussed, maybe having this as slot -1 would be better. Also we can check for overflow here.